source/appModules/audacity.py

```diff
@@ -1,15 +1,14 @@
 #appModules/audacity.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2010 NVDA Contributors <http://www.nvda-project.org/>
+#Copyright (C) 2006-2018 NVDA Contributors <https://www.nvaccess.org/>
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
 import appModuleHandler
-import winUser
 import controlTypes
 
 class AppModule(appModuleHandler.AppModule):
 
 	def event_NVDAObject_init(self,obj):
 		if obj.windowClassName=="Button" and not obj.role in [controlTypes.ROLE_MENUBAR, controlTypes.ROLE_MENUITEM, controlTypes.ROLE_POPUPMENU]:
-			obj.name=winUser.getWindowText(obj.windowHandle).replace('&','')

```

Comment:
What if someone using a really old version of Audacity decides to test this change?

Thanks.

Reply:
Same difference as if someone on XP decides to use NVDA. We no longer support them.